dataworks-common-logging.tf

```diff
@@ -0,0 +1,33 @@
+resource "github_repository" "dataworks-common-logging" {
+  name        = "dataworks-common-logging"
+  description = "Kotlin utility library to be used in Dataworks applications to ensure common logging format."
+
+  allow_merge_commit = false
+  default_branch     = "master"
+  has_issues         = true
+
```

Comment:
Add `auto-init = true` here, otherwise the pipeline will fail

Reply:
Also, I think it's wise to remove `default_branch = master` as I remember that caused problems too. The default branch gets set to master anyway when using auto_init

Reply:
Sure, I will update this. Since `default_branch = master` is also in the template I will remove it